docs/developers/creating-your-first-plugin.md

```diff
@@ -0,0 +1,101 @@
+
+This section is intended for developers who seek to develop plugins for Velocity.
```

Comment:
This lead isn't great. Can you rewrite this? It looks like you wrote a lead just because you had to write a lead. Emphasize how simple it is to get started.